test/jdk/java/lang/Class/getEnclosingClass/EnclosingClassTest.java

```diff
@@ -59,71 +48,172 @@
+        }
+    }
+
+    private void info(final Class<?> c, final Class<?> encClass, final String desc) {
+        if (!"".equals(desc)) {
             System.out.println(desc + ":");
+        }
         System.out.println(c);
         System.out.println("\tis enclosed by:\t\t" + encClass);
-        System.out.println("\thas simple name:\t`" +
-                           c.getSimpleName() + "'");
-        System.out.println("\thas canonical name:\t`" +
-                           c.getCanonicalName() + "'");
+        System.out.println("\thas simple name:\t`" + c.getSimpleName() + "'");
+        System.out.println("\thas canonical name:\t`" + c.getCanonicalName() + "'");
     }
 
-    static void match(String actual, String expected) {
-        assert((actual == null && expected == null) || actual.equals(expected));
-        System.out.println("\t`" +
-                           actual + "' matches expected `" +
-                           expected + "'");
+    private void match(final String actual, final String expected) {
+        System.out.println("actual:" + actual + "expected:" + expected);
+        assert ((actual == null && expected == null) || actual.trim().equals(expected.trim()));
+        System.out.println("\t`" + actual + "' matches expected `" + expected + "'");
     }
 
-    static void check(Class<?> c, Class<?> enc,
-                      String encName, String encNameExpected,
-                      String simpleName, String simpleNameExpected,
-                      String canonicalName, String canonicalNameExpected) {
+    private void check(final Class<?> c, final Class<?> enc,
+               final String encName, final String encNameExpected,
+               final String simpleName, final String simpleNameExpected,
+               final String canonicalName, final String canonicalNameExpected) {
         match(encName, encNameExpected);
         match(simpleName, simpleNameExpected);
         match(canonicalName, canonicalNameExpected);
     }
 
-    static void testClass(Class<?> c, TestMe annotation, Field f) {
-        if (Void.class.equals(c))
+    private void testClass(final Class<?> c, final TestMe annotation, final Field f) {
+        if (Void.class.equals(c)) {
             return;
+        }
         Class<?> encClass = c.getEnclosingClass();
         c.getEnclosingMethod(); // make sure it does not crash
         c.getEnclosingConstructor(); // make sure it does not crash
         info(c, encClass, annotation.desc());
         check(c, encClass,
-              ""+encClass, annotation.encl(),
-              c.getSimpleName(), annotation.simple(),
-              c.getCanonicalName(),
-              annotation.hasCanonical() ? annotation.canonical() : null);
-        if (void.class.equals(c))
+                "" + encClass, annotation.encl(),
+                c.getSimpleName(), annotation.simple(),
+                c.getCanonicalName(),
+                annotation.hasCanonical() ? annotation.canonical() : null);
```

Comment:
I am guessing this whitespace in line 195-197 is not intentional?  Same for line 203-206?

Reply:
Thanks, Removed in next patch. Trying to keep maximum line length 120.